src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -434,7 +434,9 @@ export function getEnvTrackingInfo( slug: string ): any {
 		for ( const key of Object.keys( envData ) ) {
 			// track doesnt like camelCase
 			const snakeCasedKey = key.replace( /[A-Z]/g, letter => `_${ letter.toLowerCase() }` );
-			result[ snakeCasedKey ] = envData[ key ];
+			const value = typeof envData[ key ] === 'object' ? JSON.stringify( envData[ key ] ) : envData[ key ];
```

Comment:
I think it's very weird that at this point the `envData[ key ]` could be a string or an object. I guess it represents a data type, in our environment data graph, which could be a string or an object. The polymorphic nature of that gives me pause, because JavaScript doesn't have super great polymorphic tools, and JSON doesn't have great ways to indicate that a node could be polymorphic, like XML does. I guess it's not a problem but it's something that we should think about.

Reply:
@pschoffer `typeof null === 'object'`, `JSON.stringify(null) === 'null'`.

Is this the intended behavior?

Reply:
I think we should enumerate the properties we care about rather than trying to handle it abstractly in this fashion

Reply:
@jesse-greathouse yeah. It is typed though, so you get all that nice doc completion (e.g. data.wordpress.type = 'image'). 

@sjinks I think that's fine. But I am also fine with adding a null check there

Reply:
Oh right we already have a list of those components so I did that. Looks better. Thanks!